app/components/progressive_image/progressive_image.js

```diff
@@ -9,6 +9,7 @@ import FastImage from 'react-native-fast-image';
 import thumb from '@assets/images/thumb.png';
 import CustomPropTypes from '@constants/custom_prop_types';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {lookupMimeType} from '@mm-redux/utils/file_utils';
```

Comment:
you should have the mimeType as part of the file info,  so no need for this here..